run_llr_template.js

```diff
@@ -74,8 +74,21 @@ switch (LLR_STEP) {
     break;
   case 7:
     var col = llr.getColForLandTrendrFromAsset(params);
+    // doesn't display the IC by default bc it slows things down so much
+    // not usually working due to memory error
+    llr.displayCollection(col);
+    // sometimes loads
+    llr.animateCollection(col);
+    // should we prompt people to go back to step 3 if some years look bad? 
```

Comment:
Please remove comments and submit as issues; one for each of those concerns, I think.